pgp/keysource.go

```diff
@@ -587,7 +566,7 @@ func fingerprintIndex(ring openpgp.EntityList) map[string]openpgp.Entity {
 // When the command fails, an error is returned.
 func gpgExec(gnuPGHome string, args []string, stdin io.Reader) (err error, stdout bytes.Buffer, stderr bytes.Buffer) {
 	if gnuPGHome != "" {
-		args = append([]string{"--no-default-keyring", "--homedir", gnuPGHome}, args...)
+		args = append([]string{"--homedir", gnuPGHome}, args...)
```

Comment:
I was confused by this argument name with the same name as new function `gnuPGHome`. Maybe rename to `gnuPGHomeDir`?

Reply:
Will actually just change it to `homeDir` to match the flag.